opennds/Makefile

```diff
@@ -30,7 +30,6 @@ define Package/opennds
   DEPENDS:=+libmicrohttpd-no-ssl
   TITLE:=open Network Demarcation Service
   URL:=https://github.com/opennds/opennds
-  CONFLICTS:=nodogsplash
```

Comment:
Okay, I found time to check out the recent discussion from here https://github.com/openwrt/routing/pull/997#issuecomment-1655227840

Unfortunately, the conflict is still required or if you want needed to have it at least in one package. Why? Both packages nodogsplash and openddns provide the same file. To be more specific:
- https://github.com/openwrt/routing/blob/967dde509b90c2166773ec672b2e8e8b81b6288a/nodogsplash/Makefile#L45
- https://github.com/openwrt/routing/blob/d038ab3d9e28a454221ce95e68b15dd3d70ed3c6/opennds/Makefile#L46

Even if you think that opkg is smart, actually, you can not have users have the option to have installed both of these packages. This can be easily reproduced and tested by using ``--force-overwrte``  to opkg. That it happens that you would have installed both packages and you would not know from which package you have that binary.

TL;DR: So, please, specify the conflict even if opkg does not allow to install it side by side, users will still be able to do that. It is always a good idea to avoid potentional issues. ;-)

Reply:
@BKPepe 
If the opennds startup checks were removed and you force installed both, then nothing bad would happen, just portal logins would not work. You could still ssh to the router etc.
However the added startup checks in opennds will prevent it from starting even if force installed.

On a non-OpenWrt Linux, where in place compiling and installing is common, the startup checks are essential, but that is not relevant here.

So in my opinion the `CONFLICTS` entry is not needed.

But if you still think it is needed, no problem putting it back in.....

Reply:
>If the opennds startup checks were removed and you force installed both, then nothing bad would happen, just portal logins would not work

I see, but still, I stand for my comment. You really don't want to have option to have installed two packages providing the same binary. If you start mixing stuff, it just calls for problems, which can be simply avoided. 

* I know, it is not your case, but it can be huge problem for SSL binaries, they do provide binary (same location, same name), but with different code base and also, the another good example is for full/small variants. That's why there are conflicts. It should be also used here. It was there, I don't see why it should be removed just now...

Reply:
The problem though is that the CONFLICTS will only be in one of them, whereas logically it should be in both, but we saw what happened when we did that. CONFLICTS does not work as expected.
So with it added back in, if opennds is installed you could install nodogsplash, if it wasn't for opkg stopping it because of ndsctl.
All this does show there is a larger issue here, with the build system.
Even with the CONFLICTS entry, you could still force-overwrite could you not?

Interesting but not worth discussing any further here.

I'll put it back in ;-)

Reply:
When there is conflict, you can not force it to have it installed. Also, one conflict is enough.  ;)

Reply:
Ok, it's back in and ci is queued....